container-chains/templates/frontier/runtime/src/lib.rs

```diff
@@ -642,6 +645,29 @@ impl pallet_hotfix_sufficients::Config for Runtime {
     type WeightInfo = pallet_hotfix_sufficients::weights::SubstrateWeight<Runtime>;
 }
 
+pub struct CanAuthor;
```

Comment:
I want to put this into` ccp-author-verifiability,` but need to think how to do it since I dont want to depend on pallet_author_inherent..

Reply:
Why would moving this to `ccp-author-verifiability` make it depend on `pallet_authorities_noting`? It only depends on `nimbus_primitives`, right?

Reply:
you need to get the authorities from somewhere. In the case of containers, this will be `pallet_authorities_noting`. In the case of tanssi, this will be `pallet_authorities_assignment`. I am thinking actually most of these traits will have to go to somewhere into the tanssi primitives, and not container-chain primitives, since Tanssi will use most of them too